Source/Core/DiscIO/DiscScrubber.cpp

```diff
@@ -21,239 +21,169 @@
-	u64 DOLSize;
-	u64 FSTOffset;
-	u64 FSTSize;
-	u32 ApploaderSize;
-	u32 ApploaderTrailerSize;
-};
-struct SPartition
+DiscScrubber::~DiscScrubber()
 {
-	u32 GroupNumber;
-	u32 Number;
-	u64 Offset;
-	u32 Type;
-	SPartitionHeader Header;
-};
-struct SPartitionGroup
-{
-	u32 numPartitions;
-	u64 PartitionsOffset;
-	std::vector<SPartition> PartitionsVec;
-};
-static SPartitionGroup PartitionGroup[4];
-
-
-void MarkAsUsed(u64 _Offset, u64 _Size);
-void MarkAsUsedE(u64 _PartitionDataOffset, u64 _Offset, u64 _Size);
-bool ReadFromVolume(u64 _Offset, u32& _Buffer, bool _Decrypt);
-bool ReadFromVolume(u64 _Offset, u64& _Buffer, bool _Decrypt);
-bool ParseDisc();
-bool ParsePartitionData(SPartition& _rPartition);
-
+}
 
-bool SetupScrub(const std::string& filename, int block_size)
+bool DiscScrubber::SetupScrub(const std::string& filename, int block_size)
 {
-	bool success = true;
-	m_Filename = filename;
-	m_BlockSize = block_size;
+	m_filename = filename;
+	m_block_size = block_size;
 
-	if (CLUSTER_SIZE % m_BlockSize != 0)
+	if (CLUSTER_SIZE % m_block_size != 0)
 	{
-		ERROR_LOG(DISCIO, "Block size %i is not a factor of 0x8000, scrubbing not possible", m_BlockSize);
+		ERROR_LOG(DISCIO, "Block size %i is not a factor of 0x8000, scrubbing not possible", m_block_size);
 		return false;
 	}
 
-	m_BlocksPerCluster = CLUSTER_SIZE / m_BlockSize;
+	m_blocks_per_cluster = CLUSTER_SIZE / m_block_size;
 
-	s_disc = CreateVolumeFromFilename(filename);
-	if (!s_disc)
+	m_disc = CreateVolumeFromFilename(filename);
+	if (!m_disc)
 		return false;
 
-	m_FileSize = s_disc->GetSize();
+	m_filesize = m_disc->GetSize();
 
-	u32 numClusters = (u32)(m_FileSize / CLUSTER_SIZE);
+	u32 num_clusters = static_cast<u32>(m_filesize / CLUSTER_SIZE);
 
 	// Warn if not DVD5 or DVD9 size
-	if (numClusters != 0x23048 && numClusters != 0x46090)
-		WARN_LOG(DISCIO, "%s is not a standard sized Wii disc! (%x blocks)", filename.c_str(), numClusters);
+	if (num_clusters != 0x23048 && num_clusters != 0x46090)
+		WARN_LOG(DISCIO, "%s is not a standard sized Wii disc! (%x blocks)", filename.c_str(), num_clusters);
 
 	// Table of free blocks
-	m_FreeTable = new u8[numClusters];
-	std::fill(m_FreeTable, m_FreeTable + numClusters, 1);
+	m_free_table.resize(num_clusters, 1);
 
 	// Fill out table of free blocks
-	success = ParseDisc();
+	const bool success = ParseDisc();
 
 	// Done with it; need it closed for the next part
-	s_disc.reset();
-	m_BlockCount = 0;
-
-	// Let's not touch the file if we've failed up to here :p
-	if (!success)
-		Cleanup();
+	m_disc.reset();
+	m_block_count = 0;
 
-	m_isScrubbing = success;
+	m_is_scrubbing = success;
 	return success;
 }
 
-size_t GetNextBlock(File::IOFile& in, u8* buffer)
+size_t DiscScrubber::GetNextBlock(File::IOFile& in, u8* buffer)
 {
-	u64 CurrentOffset = m_BlockCount * m_BlockSize;
-	u64 i = CurrentOffset / CLUSTER_SIZE;
+	u64 current_offset = m_block_count * m_block_size;
+	u64 i = current_offset / CLUSTER_SIZE;
 
 	size_t ReadBytes = 0;
```

Comment:
Since you've made all other variables use the right naming scheme, you should update the name of this one too.